multiple-sites/.env.deployer

```diff
@@ -0,0 +1,23 @@
+DB_NAME=database_name
+DB_USER=database_user
+DB_PASSWORD=database_password
+
+# Optional variables
+DB_HOST=mysql
+# DB_PREFIX=wp_
+
+WP_ENV=development
+WP_HOME=http://example.com
+WP_SITEURL=${WP_HOME}
+
+# Generate your keys here: https://roots.io/salts.html
+AUTH_KEY='generateme'
+SECURE_AUTH_KEY='generateme'
+LOGGED_IN_KEY='generateme'
+NONCE_KEY='generateme'
+AUTH_SALT='generateme'
+SECURE_AUTH_SALT='generateme'
+LOGGED_IN_SALT='generateme'
+NONCE_SALT='generateme'
+WP_CACHE_KEY_SALT='example.com:' #trailing `:` is important
```

Comment:
`WP_CACHE_KEY_SALT` should have been on a separate line with a comment explaining the purpose.

The way it looks now its part of https://roots.io/salts.html

Reply:
@iamShantanu101 please look into it

Reply:
Updated with description #18